src/components/renderer/form-loop.vue

```diff
@@ -1,18 +1,16 @@
 <template>
   <div class="form-group">
-    <div v-for="loopIndex in times" :key="loopIndex">
-      <form>
-        <vue-form-renderer
-          @submit="submit"
-          :data="getMatrixValue(loopIndex)"
-          @update="setMatrixValue(loopIndex, $event)"
-          :config="rendererConfig"
-          :computed="null"
-          :custom-css="null"
-          :watchers="null"
-        />
-      </form>
-    </div>
+    <vue-form-renderer
+      v-for="loopIndex in times"
+      :key="loopIndex"
+      :data="getMatrixValue(loopIndex)"
+      :config="rendererConfig"
+      :computed="null"
+      :custom-css="null"
+      :watchers="null"
+      @submit="submit"
+      @update="setMatrixValue(loopIndex, $event)"
+    />
```

Comment:
I didn't change any of the props here, I re-arranged them to group bindings/props data and events.